### ISSUE TITLE
复现的结果差距比较大

### ISSUE THREAD

**Issue by nightmareisme** [open-mmlab/mmdetection#9384]
### Prerequisite

- [X] I have searched [Issues](https://github.com/open-mmlab/mmdetection/issues) and [Discussions](https://github.com/open-mmlab/mmdetection/discussions) but cannot get the expected help.
- [X] I have read the [FAQ documentation](https://mmdetection.readthedocs.io/en/latest/faq.html) but cannot get the expected help.
- [X] The bug has not been fixed in the [latest version (master)](https://github.com/open-mmlab/mmdetection) or [latest version (3.x)](https://github.com/open-mmlab/mmdetection/tree/dev-3.x).

### 💬 Describe the reimplementation questions

使用coco数据集在mmdet进行复现maskrcnn的时候，最终的结果与你们得出的结果差距很大。
我尝试过不冻结训练、不使用预训练权重、还有按照你们的配置训练，结果差距都比较大。。
不过我只是用了两张卡。。会不会是gpu的个数对结果有很大的影响？？就算有影响，但是感觉差距也大了点。希望可以得到帮助

### Environment

Python: 3.7.12 | packaged by conda-forge | (default, Oct 26 2021, 06:08:21) [GCC 9.4.0]
CUDA available: True
CUDA_HOME: :/home/asc003/cuda10.2
GPU 0,1: Tesla V100-PCIE-32GB
GCC: gcc (GCC) 7.3.1 20180303 (Red Hat 7.3.1-5)
PyTorch: 1.5.1
PyTorch compiling details: PyTorch built with:
  - GCC 7.3
  - C++ Version: 201402
  - Intel(R) Math Kernel Library Version 2019.0.4 Product Build 20190411 for Intel(R) 64 architecture applications
  - Intel(R) MKL-DNN v0.21.1 (Git Hash 7d2fd500bc78936d1d648ca713b901012f470dbc)
  - OpenMP 201511 (a.k.a. OpenMP 4.5)
  - NNPACK is enabled
  - CPU capability usage: AVX2
  - CUDA Runtime 10.2
  - NVCC architecture flags: -gencode;arch=compute_37,code=sm_37;-gencode;arch=compute_50,code=sm_50;-gencode;arch=compute_60,code=sm_60;-gencode;arch=compute_61,code=sm_61;-gencode;arch=compute_70,code=sm_70;-gencode;arch=compute_75,code=sm_75;-gencode;arch=compute_37,code=compute_37
  - CuDNN 7.6.5
  - Magma 2.5.2
  - Build settings: BLAS=MKL, BUILD_TYPE=Release, CXX_FLAGS= -Wno-deprecated -fvisibility-inlines-hidden -fopenmp -DNDEBUG -DUSE_FBGEMM -DUSE_QNNPACK -DUSE_PYTORCH_QNNPACK -DUSE_XNNPACK -DUSE_INTERNAL_THREADPOOL_IMPL -O2 -fPIC -Wno-narrowing -Wall -Wextra -Werror=return-type -Wno-missing-field-initializers -Wno-type-limits -Wno-array-bounds -Wno-unknown-pragmas -Wno-sign-compare -Wno-unused-parameter -Wno-unused-variable -Wno-unused-function -Wno-unused-result -Wno-strict-overflow -Wno-strict-aliasing -Wno-error=deprecated-declarations -Wno-stringop-overflow -Wno-error=pedantic -Wno-error=redundant-decls -Wno-error=old-style-cast -fdiagnostics-color=always -faligned-new -Wno-unused-but-set-variable -Wno-maybe-uninitialized -fno-math-errno -fno-trapping-math -Werror=format -Wno-stringop-overflow, PERF_WITH_AVX=1, PERF_WITH_AVX2=1, PERF_WITH_AVX512=1, USE_CUDA=ON, USE_EXCEPTION_PTR=1, USE_GFLAGS=OFF, USE_GLOG=OFF, USE_MKL=ON, USE_MKLDNN=ON, USE_MPI=OFF, USE_NCCL=ON, USE_NNPACK=ON, USE_OPENMP=ON, USE_STATIC_DISPATCH=OFF, 

TorchVision: 0.6.0a0+35d732a
OpenCV: 4.6.0
MMCV: 1.0.5
MMDetection: ('2.3.0',)
MMDetection Compiler: GCC 7.3
MMDetection CUDA Compiler: 10.2

### Expected results

bbox_mAP: 0.3570, bbox_mAP_50: 0.5550, bbox_mAP_75: 0.3890, bbox_mAP_s: 0.2030, bbox_mAP_m: 0.3930, bbox_mAP_l: 0.4620, bbox_mAP_copypaste: 0.357 0.555 0.389 0.203 0.393 0.462, segm_mAP: 0.3170, segm_mAP_50: 0.5210, segm_mAP_75: 0.3370, segm_mAP_s: 0.1660, segm_mAP_m: 0.3510, segm_mAP_l: 0.4280, segm_mAP_copypaste: 0.317 0.521 0.337 0.166 0.351 0.428  （这是我训练得出的结果-这是不冻结层数的到的。。如果冻结了，结果比这个结果还要差点）。希望的到

R-50-FPN | pytorch | 1x | 4.4 | 16.1 |bbox 38.2 |segm 34.7  的结果

### Additional information

修改了frozen_stages=-1,norm_eval=False,和学习率=0.005（因为只有两张卡）

**Comment by ZwwWayne** [comment]
The overall batch size is important to train Mask R-CNN. If you only have two GPU cards, you should increase the batch size per GPU from 2 to 8.

**Comment by nightmareisme** [comment]
好的，我提升了GPU的批大小。效果和你们的类似。。感谢回复。。但是还有一个疑问。预训练使用的整体批量大小，会对下有任务有影响吗？ 比如说我使用两块GPU在你们的mmselfup框架上进行imagnet的预训练，但是整体的批量大小会比8张卡的小。这样的预训练权重在下游任务中（比如说maskrcnn），影响大吗？

**Comment by ZwwWayne** [comment]
If the overall batch size is not too small or too big, the performance differences of the pre-trained representations by different batch sizes in downstream tasks will be little. However, different algorithms have different properties and should be tuned and checked manually for sanity.
